tests/units/anta_tests/routing/test_bgp.py

```diff
@@ -4511,3 +4516,83 @@ def test_check_bgp_neighbor_capability(input_dict: dict[str, bool], expected: bo
         },
     },
 ]
+
+
+class TestBgpAddressFamily:
```

Comment:
Could you move these to some new models for `units/input_models/...` so we can keep sane!

Reply:
Updated. Thanks!!